tests/src/test/java/org/neo4j/graphalgo/impl/metaPathComputationTests/GetSchemaTest.java

```diff
@@ -65,7 +66,7 @@ public static void shutdownGraph() {
     }
 
     @Before
-    public void setupGraph() {
+    public void setupGraph() throws FileNotFoundException {
         graph = (HeavyGraph) new GraphLoader(api)
                 .asUndirected(true)
                 .withLabelAsProperty(true)
```

Comment:
Autoformat everywhere?

Reply:
?

Reply:
Is it each file autoformatted (key combination from previous reviews)?